skeletons/src/main/java/com/decathlon/vitamin/compose/skeletons/SkeletonMofifierExt.kt

```diff
@@ -29,3 +34,26 @@ fun Modifier.vtmnSkeleton(
             ),
         )
 }
+
+/**
+ * Extension function to apply shimmer effect on every composable
+ * @param visible visibility state of the shimmer effect
+ * @param colors The colors of the background and the highlight effect. Use [VitaminSkeletonColors]
+ * to init the expected color.
+ * @param shape shape of the skeleton
+ */
+fun Modifier.vtmnSkeleton2(
```

Comment:
Not a big fan of this naming :/

Reply:
New proposition here: https://github.com/Decathlon/vitamin-compose/pull/122#discussion_r1132416265

Reply:
Can we try with 
```
@Deprecated(
    message = "Use vtmnSkeleton2 variant with colors as mandatory parameter",
    replaceWith = ReplaceWith("vtmnSkeleton2"),
   level = DeprecationLevel.HIDDEN
)
```

And let the same naming ?